### DISCUSSION TITLE
Why Visual Studio Build Tools (and msbuild) 2022 remained x86?

### DISCUSSION THREAD

**Discussion by Thieum** [dotnet/msbuild#7114] [Q&A] [answered]
I was wondering why that specific toolchain remained x86, when the other versions of Visual Studio are now x64 ?

![image](https://user-images.githubusercontent.com/944894/144513233-216d7a2c-71a8-49a6-b6da-a3ffb25e9919.png)

**Comment by rainersigwald** [comment, marked as answer]
I suspect this was just an oversight. The Build Tools installer delivers both 32- and 64-bit MSBuild as well as 64-bit copies of for example the C++ toolchain, so it could go in either `Program Files` or `Program Files (x86)`.

I agree that it should drop the x86 now that VS itself has. Feel free to file a VS Feedback ticket for the Setup team to consider that.

**Reply by Thieum** [reply]
Noticed it while creating my own, it's already a ticket there: https://developercommunity.visualstudio.com/t/VS2022-RC3:-Default-install-location-is-/1567374

**Reply by rainersigwald** [reply]
Great! Looks like it's routed to the right folks internally.
